pkg/util/net.go

```diff
@@ -178,8 +206,11 @@ func CheckProtocol(address string) string {
 
 func AddressCount(network *net.IPNet) float64 {
 	prefixLen, bits := network.Mask.Size()
-	if bits-prefixLen < 2 {
-		return 0
+	// Special case handling for /31 and /32 subnets
+	if bits-prefixLen == 1 {
+		return 2 // /31 subnet
+	} else if bits-prefixLen == 0 {
+		return 1 // /32 subnet
```

Comment:
Why should address count be `1` for `/32` or `/128` networks? When and where will a `/32` or `/128` be used?

Reply:
I have added it for the util function as well, IMO it would be good to validate against this case, as I think it won't be used in kube-ovn subnets. wdyt?